src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -10,6 +10,9 @@
 
 from ._client_factory import web_client_factory
 
+def output_slots_In_table(slots):
+    return [{'name': s['name'], 'status': s['state'], 'app service plan': s['appServicePlan']} for s in slots]
```

Comment:
Haven't seen any other table headings with spaces in.
'app service plan' -> 'appServicePlan'?